src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -447,6 +447,135 @@ JSON example:
+| EchoTime                          | REQUIRED                                                                                                                        | [number][] or [array][] of [numbers][]  | The echo time (TE) for the acquisition, specified in seconds. Specify either one number for the total time-series, or provide an array of numbers with different values of each volume. Corresponds to DICOM Tag 0018, 0081 `Echo Time` (please note that the DICOM term is in milliseconds not seconds).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
+| SliceTiming                       | RECOMMENDED, but REQUIRED in case of a 2D acquisition, or when VolumeTiming is defined                                          | [array][] of [numbers][]                | The time at which each slice was acquired within each volume (frame) of the acquisition with respect to the beginning of volume acquisition, namely after the PostLabelingDelay. In the case of 3D acquisition, a zero is entered. The length of this array of numbers should be equal to the number of slices for 2D acquisition, with the first time being zero, effectively equal to the PostLabelingDelay. Slice timing is not the same as slice order. Rather, it is a list of times containing the time (in seconds) of each slice acquisition in relation to the beginning of volume acquisition. The list goes through the slices along the slice axis in the slice encoding dimension. Note that to ensure the proper interpretation of the `SliceTiming` field, it is important to check if the OPTIONAL `SliceEncodingDirection` exists. In particular, if `SliceEncodingDirection` is negative, the entries in `SliceTiming` are defined in reverse order with respect to the slice axis, such that the final entry in the `SliceTiming` list is the time of acquisition of slice zero. |
```

Comment:
```suggestion
| SliceTiming                       | RECOMMENDED, but REQUIRED in case of a 2D acquisition, or when `VolumeTiming` is defined                                          | [array][] of [numbers][]                | The time at which each slice was acquired within each volume (frame) of the acquisition with respect to the beginning of volume acquisition, namely after the `PostLabelingDelay`. In the case of 3D acquisition, a zero is entered. The length of this array of numbers should be equal to the number of slices for 2D acquisition, with the first time being zero, effectively equal to the `PostLabelingDelay`. Slice timing is not the same as slice order. Rather, it is a list of times containing the time (in seconds) of each slice acquisition in relation to the beginning of volume acquisition. The list goes through the slices along the slice axis in the slice encoding dimension. Note that to ensure the proper interpretation of the `SliceTiming` field, it is important to check if the OPTIONAL `SliceEncodingDirection` exists. In particular, if `SliceEncodingDirection` is negative, the entries in `SliceTiming` are defined in reverse order with respect to the slice axis, such that the final entry in the `SliceTiming` list is the time of acquisition of slice zero. |
```